File: server/src/main/java/com/mobilebcs/controller/user/UserType.java
package com.mobilebcs.controller.user;

import java.util.Arrays;

public enum UserType {

    QUALIFIER,
    VIEWER;

    public static UserType valueFrom(String userType) {
        return Arrays.stream(values()).filter(value->value.name().equals(userType)).findAny().orElse(null);
    }
}

File: calificator/lib/src/user/user_type.dart
enum UserType{
  qualifier,
  viewer
}


extension UserTypeExtension on UserType {

  String get value {
    switch (this) {
      case UserType.qualifier:
        return 'Calificador';
      case UserType.viewer:
        return 'Observador';
    }
  }

}

